userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -204,7 +204,7 @@ Norway Spruce Picea abies shoot with foliage buds.
 ```
 
 {{< imgproc spruce Fill "400x450" >}}
-Norway Spruce Picea abies shoot with foliage buds.
+Norway Spruce <i>Picea abies</i> shoot with foliage buds.
```

Comment:
Consider making the same update to the `go-html-template` above.

Reply:
Good point. I made the same change to the template meanwhile.